rust/Cargo.toml

```diff
@@ -32,14 +32,14 @@ rand = "0.7"
 rand_chacha = "0.2.1"
 rayon = "1.2.1"
 anyhow = "1.0.23"
-bellperson = { version = "0.14.1", default-features = false }
+bellperson = { version = "0.16", default-features = false }
 serde_json = "1.0.46"
 rust-gpu-tools = "0.3.0"
```

Comment:
`rust-gpu-tools` should also be updated to the version `bellperson` is using.

Reply:
Good catch, thanks